client/web/src/Structure/Admin/State/action.js

```diff
@@ -250,5 +409,10 @@ export const adminActions = {
     updateCrudTopicsRequest,
     updateCrudTopicsSuccess,
     updateCrudTopicsFailure,
-    updateCrudTopics
+    updateCrudTopics,
+    getQuestionsRequest,
+    getTopicsRequest,
+    getQuestionsByTopicRequest,
+    deleteQuestionsRequest,
+    addQuestionsRequest
```

Comment:
export only required function, the failure, success, request. it won't be used anywhere, remove unnecessay from object

Reply:
This is actually done by @Anantjain226